src/Core_Command.php

```diff
@@ -374,6 +374,9 @@ public function is_installed( $args, $assoc_args ) {
 	 * --admin_email=<email>
 	 * : The email address for the admin user.
 	 *
+	 * [--locale=<locale>]
+	 * : The locale/language for the installation (e.g. `de_DE`). Default is empty.
```

Comment:
I think it makes more sense to state what the fallback locale is as default here.
```suggestion
	 * : The locale/language for the installation (e.g. `de_DE`). Default is `en_US`.
```